packages/toolpad-core/src/DataGrid/DataGrid.tsx

```diff
@@ -0,0 +1,1041 @@
+'use client';
```

Comment:
I have been putting the `use client` directive in the index file only.
But we can change it for the other components if this works better.

Reply:
Yes, let's keep them at the smallest scope possible. Probbaly even debatable if libraries should have them at all